ui/src/VoxettaApp.js

```diff
@@ -29,13 +38,17 @@ export class VoxettaApp extends LitElement {
 
     constructor() {
         super();
-        this.state = 'RECORD-PAGE'; 
+        this.cookieService = new CookieService();
+        this.state = States.RECORD_PAGE; 
+        this.userId = this.cookieService.getCookieValue("userId");
```

Comment:
suggestion, cookie service should provide getters for each supported field.

such as:

```javascript
this.cookieService.getUserId();
this.cookieService.getGender();
this.cookieService.getUserAge();
```

So the consumer of cookie service doesn't need to know the exact names of the fields

Reply:
adjusted